Digipost/APIClient.swift

```diff
@@ -427,9 +418,9 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
         let baseUri = __SERVER_URI__
         let completeUri = "\(baseUri)\(uri)"
         let task = self.urlSessionTaskWithNoAuthorizationHeader(httpMethod.post, url: completeUri, parameters: parameters, success: { () -> Void in
-            DLog("successfully sent log")
+            DLog("Successfully sent: \(parameters)")
```

Comment:
Suggest we use something slightly more descriptive like "Successfully sent log with parameters …"